buildenv/jenkins/aqaTestPipeline.groovy

```diff
@@ -86,7 +87,12 @@ timestamps {
                 }
             }
         } else {
-            generateJobs(JDK_VERSION, TEST_FLAG, PLATFORMS, TARGETS, PARALLEL)
+            if ( MODE == 'RELAY' ) {
+                remoteTriggerTemurinJCK()
```

Comment:
Eventually may want to rename to remoteTriggerPrivateJCK() and give the parameterized option of where the privately run TCKs are triggered... future iteration.